Steps/Steps.module.less

```diff
@@ -16,18 +16,17 @@
 			font-weight: @sand-steps-step-number-font-weight;
 		}
 
-		&.dots {
-			font-size: @sand-steps-step-dot-font-size;
+		&.highlightCurrentOnly {
+			font-size: @sand-steps-step-highlight-current-only-font-size;
 			opacity: @sand-steps-step-past-opacity;
-			margin: 0;
-			width: @sand-steps-step-width;
+			margin: @sand-steps-step-highlight-current-only-margin;
 
 			&.current {
-				font-size: @sand-steps-step-current-dot-font-size;
+				font-size: @sand-steps-step-highlight-current-only-current-font-size;
 			}
 
 			&.future {
-				opacity:  @sand-steps-step-past-opacity;
+				opacity: @sand-steps-step-past-opacity;
```

Comment:
How about renaming this to `sand-steps-step-highlight-current-only-future-opacity` which has 1 as the value?

Reply:
I renamed to `sand-steps-step-highlight-current-only-future-opacity`.